misk-config/src/main/kotlin/misk/config/ConfigModule.kt

```diff
@@ -11,11 +11,11 @@ class ConfigModule<T : wisp.config.Config>(
   override fun configure() {
     install(AppNameModule(appName))
     bind(configClass).toInstance(config)
-    bind<wisp.config.Config>().toInstance(config)
```

Comment:
This is probably a breaking change if there are existing callsites depending on the wisp config on their injection? If we go update all callsites promptly we can still land it as is.